assembl/static2/js/app/reducers/postsReducer.js

```diff
@@ -29,38 +38,58 @@ export const topPostBody = (state = '', action) => {
   }
 };
 
-export const topPostFormStatus = (state = false, action) => {
+export const subjectTopPostRemainingChars = (state = 10000, action) => {
   switch (action.type) {
-  case 'UPDATE_TOP_POST_FORM_STATUS':
-    return action.isTopPostFormActive;
+  case 'UPDATE_TOP_POST_SUBJECT_REMAINING_CHARS':
+    return action.subjectTopPostRemainingChars;
   default:
     return state;
   }
 };
 
-export const subjectRemainingChars = (state = 10000, action) => {
+export const bodyTopPostRemainingChars = (state = 10000, action) => {
   switch (action.type) {
-  case 'UPDATE_SUBJECT_REMAINING_CHARS':
-    return action.subjectRemainingChars;
+  case 'UPDATE_TOP_POST_BODY_REMAINING_CHARS':
+    return action.bodyTopPostRemainingChars;
   default:
     return state;
   }
 };
 
-export const bodyRemainingChars = (state = 10000, action) => {
+export const answerPostFormStatus = (state = false, action) => {
   switch (action.type) {
-  case 'UPDATE_BODY_REMAINING_CHARS':
-    return action.bodyRemainingChars;
+  case 'UPDATE_ANSWER_POST_FORM_STATUS':
+    return action.isAnswerPostFormActive;
   default:
     return state;
   }
 };
 
+export const answerPostBody = (state = '', action) => {
```

Comment:
I would make the same comment as previous PR: https://github.com/assembl/assembl/pull/121#discussion_r126692963 . How about the solution I proposed? (a better name for basicActionFactory could be basicReducerFactory)